mmv1/products/bigqueryreservation/CapacityCommitment.yaml

```diff
@@ -26,11 +26,8 @@ references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
     'Introduction to Reservations': 'https://cloud.google.com/bigquery/docs/reservations-intro'
   api: 'https://cloud.google.com/bigquery/docs/reference/reservations/rest/v1/projects.locations.capacityCommitments'
-id_format: 'projects/{{project}}/locations/{{location}}/capacityCommitments/{{capacity_commitment_id}}'
-import_format:
-  [
-    'projects/{{project}}/locations/{{location}}/capacityCommitments/{{capacity_commitment_id}}',
-  ]
+id_format: '{{name}}'
+import_format: ['{{name}}']
```

Comment:
I don't think it will fix the bug, as `name` is the same with `'projects/{{project}}/locations/{{location}}/capacityCommitments/{{capacity_commitment_id}}'`

Reply:
Can you please try the recommendation in the [comment](https://github.com/hashicorp/terraform-provider-google/issues/16204#issuecomment-1760418642)?